elasticsearch/includes/classes/class-health.php

```diff
@@ -79,6 +79,15 @@ public static function validate_index_entity_count( array $query_args, \ElasticP
 	 */
 	public static function validate_index_users_count() {
 		$users = Indexables::factory()->get( 'user' );
+		// Indexables::factory()->get() returns boolean|array
+		// False is returned in case of error
+		if ( ! $users ) {
+			$result = [
```

Comment:
What do you think about returning a new WP_Error here instead? That would be more in-line with other error handling here (instead of having 2 different types of error responses).

Reply:
I am returning a `result` here instead of `WP_Error` because that's what the function already did: see a few lines below where in case `self::validate_entity_count` returns an error, we are returning the error message inside of the result.
I suggest we change the behavior in both cases then?

Reply:
I have made these changes in 408cab96.